CONTRIBUTORS_QUICK_START.rst

```diff
@@ -302,18 +302,27 @@ Setting up Breeze
 5. When you enter Breeze environment you should see prompt similar to ``root@e4756f6ac886:/opt/airflow#``. This
    means that you are inside the Breeze container and ready to run most of the development tasks. You can leave
    the environment with ``exit`` and re-enter it with just ``breeze`` command.
-   Once you enter breeze environment, create airflow tables and users from the breeze CLI. ``airflow db reset``
+
+.. code-block:: bash
+
+  root@b76fcb399bb6:/opt/airflow# airflow db init
```

Comment:
```suggestion
  root@b76fcb399bb6:/opt/airflow# airflow db reset
```

The `init` command is deprecated. Reset is the right one.

Reply:
Not resolved

Reply:
And sorry, partly my fault - initial review from me was to change from `reset`to `init` - did not know that init is deprecated :-(

Reply:
No issue